crates/scroll/cli/src/commands.rs

```diff
@@ -0,0 +1,41 @@
+use crate::ScrollChainSpecParser;
+use clap::Subcommand;
+use reth_cli::chainspec::ChainSpecParser;
+use reth_cli_commands::{
+    config_cmd, db, dump_genesis, init_cmd, node, node::NoArgs, p2p, prune, recover, stage,
+};
+use std::fmt;
+
+/// Commands to be executed
+#[derive(Debug, Subcommand)]
+pub enum Commands<
+    Spec: ChainSpecParser = ScrollChainSpecParser,
+    Ext: clap::Args + fmt::Debug = NoArgs,
+> {
+    /// Start the node
+    #[command(name = "node")]
+    Node(Box<node::NodeCommand<Spec, Ext>>),
+    /// Initialize the database from a genesis file.
+    #[command(name = "init")]
+    Init(init_cmd::InitCommand<Spec>),
+    /// Dumps genesis block JSON configuration to stdout.
+    DumpGenesis(dump_genesis::DumpGenesisCommand<Spec>),
+    /// Database debugging utilities
+    #[command(name = "db")]
+    Db(db::Command<Spec>),
+    /// Manipulate individual stages.
+    #[command(name = "stage")]
+    Stage(Box<stage::Command<Spec>>),
+    /// P2P Debugging utilities
+    #[command(name = "p2p")]
+    P2P(p2p::Command<Spec>),
+    /// Write config to stdout
+    #[command(name = "config")]
+    Config(config_cmd::Command),
+    /// Scripts for node recovery
+    #[command(name = "recover")]
+    Recover(recover::Command<Spec>),
+    /// Prune according to the configuration without any limits
+    #[command(name = "prune")]
+    Prune(prune::PruneCommand<Spec>),
+}
```

Comment:
question: Had you considered using the `Commands` type from the `reth` crate?

Reply:
I have been checking, it's possible to use them, but this means removing the scroll feature gate on the `reth` crate, which in turn means we need to fix a certain amount of compilation issues. I feel like it's not worth our time for a couple of commands, what do you think?